### PR TITLE
Schedule post submits for fusion engine

### PR DESCRIPTION
**Pull request by jtmcdole**
1. Collect post submits from the engine repo if its fusion
2. When scheduling; set the property 'is_fusion'

### CODE REVIEW

**Comment by jtmcdole** [comment]
i'm going to find the other TODOs around post submit and resolve

**Comment by jtmcdole** [comment]
Cocoon app and auto_submit are currently not deployable:

https://pantheon.corp.google.com/cloud-build/builds;region=global/6abd3141-5827-4f46-a9ff-3e912e7fcb65;step=1?inv=1&invt=AbjPcg&project=flutter-dashboard

https://pantheon.corp.google.com/cloud-build/builds;region=global/09e71c1d-0837-4a67-8392-55c02b3de873;step=0?inv=1&invt=AbjPcg&project=flutter-dashboard

File: app_dart/lib/src/service/luci_build_service.dart [hunk 1]
```diff
@@ -524,6 +524,9 @@ class LuciBuildService {
     final isFusion = await fusionTester.isFusionBasedRef(slug, sha);
     if (isFusion) {
       properties['is_fusion'] = 'true';
+      // When in fusion, we want the recipies to override the engine.realm
```
Comment (christopherfujino) [inline app_dart/lib/src/service/luci_build_service.dart:527]:
```suggestion
      // When in fusion, we want the recipes to override the engine.realm
```

**Review by christopherfujino — APPROVED** [review]
lgtm

**Review by yjbanov — APPROVED** [review]
lgtm

**Pull request merged**